store/src/java/com/zimbra/cs/mailbox/ContactBackupThread.java

```diff
@@ -64,26 +66,19 @@ public static synchronized void startup() {
                 ZimbraLog.contactbackup.warn("can not start another thread");
                 return;
             }
-            setContactBackupFrequency();
-            if (contactBackupFreq == 0) {
-                ZimbraLog.contactbackup.warn("%s is set to 0", Provisioning.A_zimbraFeatureContactBackupFrequency);
-                return;
-            }
-            if (!CallbackUtil.logStartup(Provisioning.A_zimbraFeatureContactBackupFrequency)) {
-                return;
-            }
             backupThread = new ContactBackupThread();
+            shutdownRequested = false;
             backupThread.start();
         }
     }
 
     public static synchronized void shutdown() {
```

Comment:
Why the use of synchronized method and synchronized block

Reply:
done